cmd/federated.go

```diff
@@ -33,7 +33,7 @@ func init() {
 	federatedCmd.PersistentFlags().StringVarP(&federatedFile,
 		"file",
 		"f",
-		cfgDir+"/federated.yaml",
+		"/federated.yaml",
```

Comment:
I haven't checked this, but it seems surprising that it works? Looks like it's saying that the default for a federated file is ./federated.yaml? 

In fact, I'm not sure we even need this ability to override the file name. We could keep things simpler by omitting these flags (for master, node & federated file locations) - wdyt? We could take those out under a separate issue though if you agree.